.teamcity/_self/projects/WebApp.kt

```diff
@@ -235,6 +235,19 @@ object BuildDockerImage : BuildType({
 						</tr>
 					</table>
 				</details>
+				<details>
+					<summary>Automattic for Agencies live <a href="https://calypso.live?image=registry.a8c.com/calypso/app:build-%build.number%&env=a8c-for-agencies">(direct link)</a></summary>
```

Comment:
An opportunity for improvement is to use a loop for those. Kotlin supports them and it would allow for shortening the code by avoiding repetition, easier maintenance, and quicker addition of a new environment in the future.